tests/unit/readers/cubeconfig/v1_0/test_cube_from_config_json.py

```diff
@@ -754,5 +752,147 @@ def test_observation_value_data_type_extraction():
+    """
+    data = pd.DataFrame({"The Column": ["1", "2", "3"]})
+
+    (column, _) = _get_qb_column_from_json(
+        {
+            "type": "attribute",
+            "describes_observations": "Obs Val Column Title",
+            "data_type": "integer",
+            "from_existing": "http://example.com/attributes/some-existing-attribute",
+            "required": False,
+        },
+        "The Column",
+        data,
+        4,
+    )
+
+    assert isinstance(
+        column.structural_definition, ExistingQbAttributeLiteral
+    ), column.structural_definition
+    column_title = column.structural_definition.observed_value_col_title
+    assert column_title == "Obs Val Column Title"
+
+
+def test_describes_obs_val_existing_attribute_resource_column():
+    """
+    Ensure that the `describes_observations` property value is passed to a column which represents a ExistingQbAttribute.
+    """
+    data = pd.DataFrame({"The Column": ["1", "2", "3"]})
+
+    (column, _) = _get_qb_column_from_json(
+        {
+            "type": "attribute",
+            "from_existing": "http://example.com/attributes/some-existing-attribute",
+            "values": False,
+            "cell_uri_template": "http://qudt.com/units/{+the_column}",
+            "describes_observations": "Obs Val Column Title",
+        },
+        "The Column",
+        data,
+        4,
+    )
+
+    assert isinstance(
+        column.structural_definition, ExistingQbAttribute
+    ), column.structural_definition
+    column_title = column.structural_definition.observed_value_col_title
+    assert column_title == "Obs Val Column Title"
+
+
+def test_describes_obs_val_new_attribute_literal_column():
+    """
+    Ensure that the `describes_observations` property value is passed to a column which represents a NewQbAttributeLiteral.
+    """
+    data = pd.DataFrame({"The Column": ["1", "2", "3"]})
+
+    (column, _) = _get_qb_column_from_json(
+        {
+            "type": "attribute",
+            "label": "Yay",
+            "data_type": "decimal",
+            "describes_observations": "Obs Val Column Title",
+        },
+        "The Column",
+        data,
+        4,
+    )
+
+    assert isinstance(
+        column.structural_definition, NewQbAttributeLiteral
+    ), column.structural_definition
+    column_title = column.structural_definition.observed_value_col_title
+    assert column_title == "Obs Val Column Title"
+
+
+def test_describes_obs_val_new_attribute_resource_column():
+    """
+    Ensure that the `describes_observations` property value is passed to a column which represents a NewQbAttribute.
+    """
+    data = pd.DataFrame({"The Column": ["1", "2", "3"]})
+
+    (column, _) = _get_qb_column_from_json(
+        {"type": "attribute", "describes_observations": "Obs Val Column Title"},
+        "The Column",
+        data,
+        4,
+    )
+
+    assert isinstance(
+        column.structural_definition, NewQbAttribute
+    ), column.structural_definition
```

Comment:
Could the assertion statement be expanded to show why it may fail. E.g. Expected `QbMultiUnits`, got `NewQbAttribute`

Reply:
After discussion, this can be resolved without changes. All relevant information can be ascertained already without adding a maintenance burden.